trl/trainer/ppo_trainer.py

```diff
@@ -589,6 +622,10 @@ def collator(data):
         if self.lr_scheduler is not None:
             self.lr_scheduler.step()
 
+        self.current_step += 1
+        if self.config.save_steps and self.current_step % self.config.save_steps == 0:
+            self.accelerator.save_state()
```

Comment:
Nice! Does this needs to be executed on the main rank in case of distributed training? Maybe the `accelerator` already takes care of that